src/mcts/node.h

```diff
@@ -206,33 +206,42 @@ class Node {
   std::string DebugString() const;
 
  private:
-  // List of edges.
+  // Turns out the alignment and padding of the Node was costing more than 8
+  // bytes on modern compilers. Simply rearranging the members leads to
+  // significant savings on padding. As such, instead of being organized
+  // conceptually, the members are merely ordered from largest to smallest.
+
+  // TODO: shrink the padding on this somehow? It takes 16 bytes even though
+  // only 10 are real! Maybe even merge it into this class??
   EdgeList edges_;
-  // Index of this node is parent's edge list.
-  uint16_t index_;
+
+  // Pointer to a parent node. nullptr for the root.
```

Comment:
```cpp
// 8-byte fields
```
(I think you had that initially)

Reply:
I did, but it seemed slightly unnecessary in combination with other comments from you about unnecessary comments. I guess it doesn't hurt to have them